Simplified/MyBooks/NYPLMyBooksDownloadCenter.m

```diff
@@ -258,6 +266,23 @@ - (void)URLSession:(__attribute__((unused)) NSURLSession *)session
            userID:[[NYPLUserAccount sharedAccount] userID]
            deviceID:[[NYPLUserAccount sharedAccount] deviceID]];
         }
+#endif
+        break;
+      }
+      case NYPLMyBooksDownloadRightsManagementLCP: {
+#if defined(LCP)
+        LCPLibraryService *lcpService = [[LCPLibraryService alloc] init];
+        [lcpService fulfill:tmpSavedFileURL completion:^(NSURL *localUrl, NSURLSessionDownloadTask *downloadTask, NSError *error) {
```

Comment:
No it is not synchronous, but we can't lose completion invocation in this case - it will be called anyway. We can literally set `lcpService = nil;` after `fulfill` and still get the completion block called.

I checked URLSession:downloadTask:didFinishDownloadingToURL and URL parameter is not marked as _Nullable, swift version of it is not nullable either.